public/app/components/admin.dash/system/system-manager.tpl.html

```diff
@@ -24,13 +24,6 @@
             <input type="text" ng-model="settingsGlobal.jargon.en.motions">
           </md-input-container>
 
-        <div class="md-subheader">Email Notifications when a motion changes</div>
-          <md-checkbox ng-true-value="1" ng-false-value="0" ng-model="settingsGlobal.motion.email.admin">
```

Comment:
add `ng-hide="true"` instead of removing the code, this way when it actually works again we don't have to re-add the code in.